Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -611,7 +611,8 @@ def setup_(self, step, stepName, stepDict, k, properties):
         '--procModifiers': 'pixelNtupletFit,gpuValidation'
     },
     harvest = {
-        '-s': 'HARVESTING:@trackingOnlyValidation+@pixelTrackingOnlyDQM'
+        '-s': 'HARVESTING:@trackingOnlyValidation+@pixelTrackingOnlyDQM',
+        '--procModifiers': 'gpuValidation'
```

Comment:
Could you add the same also to the `.583` and `.587` workflows, and check if the new DQM works in the `.593` and `.597` workflows ?

Reply:
done in [b52348d](https://github.com/cms-sw/cmssw/pull/37619/commits/b52348d3f6ed40968459953a29cf6767a2e88c7b).
Testing now.

Reply:
```
11634.503_TTbar_14TeV+2021_Patatrack_PixelOnlyGPU_Validation+TTbar_14TeV_TuneCP5_GenSim+Digi+RecoNano+HARVESTNano Step0-PASSED Step1-PASSED Step2-PASSED Step3-PASSED  - time date Fri Apr 22 15:07:54 2022-date Fri Apr 22 14:56:55 2022; exit: 0 0 0 0
11634.506_TTbar_14TeV+2021_Patatrack_PixelOnlyTripletsGPU+TTbar_14TeV_TuneCP5_GenSim+Digi+RecoNano+HARVESTNano Step0-PASSED Step1-PASSED Step2-PASSED Step3-PASSED  - time date Fri Apr 22 15:07:54 2022-date Fri Apr 22 14:56:55 2022; exit: 0 0 0 0
11634.583_TTbar_14TeV+2021_Patatrack_AllGPU_Validation+TTbar_14TeV_TuneCP5_GenSim+Digi+RecoNano+HARVESTNano Step0-PASSED Step1-PASSED Step2-PASSED Step3-PASSED  - time date Fri Apr 22 15:07:59 2022-date Fri Apr 22 14:56:56 2022; exit: 0 0 0 0
11634.587_TTbar_14TeV+2021_Patatrack_AllTripletsGPU_Validation+TTbar_14TeV_TuneCP5_GenSim+Digi+RecoNano+HARVESTNano Step0-PASSED Step1-PASSED Step2-PASSED Step3-PASSED  - time date Fri Apr 22 15:07:59 2022-date Fri Apr 22 14:56:56 2022; exit: 0 0 0 0
4 4 4 4 tests passed, 0 0 0 0 failed
```

looks good.